osu.Game/Database/ArchiveModelManager.cs

```diff
@@ -160,7 +160,8 @@ protected async Task Import(ProgressNotification notification, params string[] p
 
                     lock (imported)
                     {
-                        imported.Add(model);
+                        if (model != null)
```

Comment:
was the model ever null before this change?

Reply:
No, but now it will be null for the .osg files in the Data/r directory. Should I handle it differently?

Reply:
yes, please avoid adding new cases of null where they didn't exist before. especially if it's a common occurrence (it shouldn't happen in the first place) and not an outlier.